### PR TITLE
kit tests in make test

### PR DESCRIPTION
**Pull request by kegl**

### CODE REVIEW

**Comment by agramfort** [comment]
LGTM

File: .travis.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ before_install:
     - export PATH=/home/travis/miniconda/bin:$PATH
     - conda update --yes --quiet conda
 install:
+    - pip install codecov
```
Comment (agramfort) [inline .travis.yml]: do the pip install after the conda install

File: .travis.yml [hunk 2]
```diff
@@ -6,6 +6,7 @@ before_install:
...
     - conda create -n testenv --yes pip python=3.6
     - source activate testenv
     - conda install --yes --quiet numpy scipy matplotlib scikit-learn pandas nose
```
Comment (agramfort) [inline .travis.yml]: pip install codecov

here

File: .travis.yml [hunk 3]
```diff
@@ -17,12 +18,14 @@ install:
     - git clone https://github.com/ramp-kits/iris.git
     - cd ..
 script:
-    - nosetests rampwf/tests
+    - nosetests --with-coverage rampwf/tests
     - cd kits/boston_housing
```
Comment (agramfort) [inline .travis.yml]: don't move around with cd as coverage will be put the current dir so it will not be in a single place

Comment (kegl) [inline .travis.yml]: done

**Pull request merged**